app/assets/javascripts/discourse/app/components/search-menu/results.js

```diff
@@ -50,6 +62,7 @@ export default class Results extends Component {
     return this.search.results.grouped_search_result?.search_log_id;
   }
 
+  // TODO: unused action `updateSearchTopics`
```

Comment:
I would just delete this and the tracked prop if they seem unused ( I can't find any use either), probably a mistake in this giant PR https://github.com/discourse/discourse/commit/a2b038ffe7043b6e725c75591dd157c98015c26c